augly/image/functional.py

```diff
@@ -1786,3 +1786,64 @@ def vflip(
     imutils.get_metadata(metadata=metadata, function_name="vflip", **func_kwargs)
 
     return imutils.ret_and_save_image(aug_image, output_path)
+
+
+def skew(
+    image: Union[str, Image.Image],
+    output_path: Optional[str] = None,
+    metadata: Optional[List[Dict[str, Any]]] = None,
+    level: float, 
+    max_level,
+    axis: str
+) -> Image.Image:
+    
+    """
+    Skew image with respect to X-axis or Y-axis
+    
+    @param image: the path to an image or a variable of type PIL.Image.Image
+        to be augmented
+        
+    @param level: Level of the operation that will be in [0, `PARAMETER_MAX`]
+    
+    @param maxval: Maximum value that the operation can have. This will be 
+                scaled to level/PARAMETER_MAX.
```

Comment:
```suggestion
    @param max_level: Maximum value that the operation can have. This will be
        scaled to level / PARAMETER_MAX.
```

Reply:
I'm not sure what you mean by this comment. I don't see you scaling the `level` in the code so that this is the max value. Instead it looks like you're multiplying by `max_level / 10`. What is the intended purpose of this param?